translations/CONTRIBUTING.portuguese_br.md

```diff
@@ -0,0 +1,153 @@
+
+<a name="first-timers"></a>
+
+## 3. Boas-vindas aos contribuidores de primeira viagem!
+
+Nós apreciamos contribuidores de primeira viagem e ficaremos feliz de ajudá-lo na sua iniciação. Em caso de dúvidas, contate-nos!
+
+Você encontrará issues (problemas) apropriadas para novos contribuidores [aqui](https://github.com/hasura/graphql-engine/issues?q=is%3Aopen+is%3Aissue+label%3A%22good+first+issue%22).
+
+<a name="areas"></a>
+
+## 4. Áreas para Contribuição
+
+É claro, adoramos contribuições para todos os componentes do Hasura. Porém, identificamos 3 áreas que se encaixam perfeitamente para contribuições open source.
+
+### Documentação
+
+Nossa meta é manter nossa documentação compreensiva e atualizada. Se você gostaria de nos ajudar com isso, somos gratos por qualquer tipo de contribuição:
+
+- Reportar conteúdo faltante
+
+- Corrigir erros em documentação já existente
+
+- Nos ajudar adicionando nova documentação
+
+O guia de contribuição para documentação pode ser encontrado em [docs/CONTRIBUTING.md](docs/CONTRIBUTING.md) (em inglês).
+
+### Conteúdo da comunidade
+
+Desde que lançamos nossa [página de aprendizado](https://hasura.io/learn/), nós estamos felizes com contribuições que:
+
+- Corrigem erros em tutoriais de aprendizado já existentes
+
+- Adicionam novos tutoriais (por favor, nos contate se você tem ideias de como evitar duplicação de palavras)
+
+O README do repositório de aprendizado pode ser encontrado [aqui](https://github.com/hasura/learn-graphql) (em inglês).
+
+Além do conteúdo de aprendizado, identificamos três outras formas de contribuir com conteúdo técnico para a comunidade:
+
+- [Boilerplates](https://github.com/hasura/graphql-engine/tree/master/community/boilerplates)
+
+- [Exemplos de apps](https://github.com/hasura/graphql-engine/tree/master/community/sample-apps)
+
+- [Ferramentas](community/tools)
+
+Se você gostaria de contribuir para a comunidade:
+
+- escrevendo um post técnico para um blog;
+
+- palestrar em um evento;
+
+- organizar um workshop;
+
+cheque nossa [wiki da comunidade](https://github.com/hasura/graphql-engine/wiki/Community-Wiki) (em inglês).
+
+Sinta-se à vontade para criar um pull request se você possui algo para adicionar, mesmo que não seja relacionado a algo mencionado acima.
+
+### CLI Hasura
+
+Nós temos algumas issues (problemas) na nossa CLI (Interface de Linha de Comando) que se encaixam em contribuições open source. Se você sabe Go, ou gostaria de aprender na prática, cheque as seguintes [issues](https://github.com/hasura/graphql-engine/issues?q=is%3Aopen+is%3Aissue+label%3Ac%2Fcli+label%3A%22help+wanted%22).
+
+O README do repositório da CLI pode ser encontrado [aqui](https://github.com/hasura/graphql-engine/tree/master/cli) (em inglês).
+
+<a name="ways"></a>
+
+## 5. Maneiras de contribuir
+
+### Criando uma issue
+
+- Tenha certeza de que você testou com a última versão lançado. É possível que já tenhamos corrigido o bug que você está enfrentando.
```

Comment:
Biggest proof of Google Translator: not knowing how to put masculine or feminine pronoun.

- Tenha certeza de que você testou com a última versão lançada. É possível que já tenhamos corrigido o bug que você está tentando consertar.